start-all.sh

```diff
@@ -26,6 +26,17 @@ else
     GROUPID=`id -g`
     USER_PERMISSIONS="-u $UID:$GROUPID"
 fi
+
+group_args=()
+is_podman="$(docker --help | grep -o podman)"
+if [ ! -z "$is_podman" ]; then
+    for gid in $(id -G); do
+        group_args+=(--group-add "$gid")
+    done
+    group_args+=(--userns=keep-id)
+fi
```

Comment:
Ok, I'm happy to report that this change (plus the one line below using group_args) is what solves the problem I had with the "-d" option, where the container didn't have permision to actual use the directory I gave it because it belonged to my user. Please split this as a separate patch, explain why it's needed, and then I'll commit this.
But, to be honest, while this works, I don't understand what that "group-add" is supposed to be doing... Isn't "--userns=keep-id" all the magic we need? What is this group-add thing doing?